### ISSUE TITLE
What is the loss_weights['reconstruction'] during training?

### ISSUE THREAD

**Issue by Guanyuansheng** [Songluchuan/AdaSR-TalkingHead#3]
This weight seems to be missing from the config file

**Comment by Songluchuan** [comment]
I follow the face2vid to finish my loss function, please refer to the https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis for details.

**Comment by Guanyuansheng** [comment]
> I follow the face2vid to finish my loss function, please refer to the https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis for details.

As far as I know, L1 loss is not used in the facevid2vid. I would like to ask what benefits will be brought by adding L1 loss, and what is the weight of L1 loss?

**Comment by Songluchuan** [comment]
The L1 is from the inspiration of ESRGAN, we use it together with perceptual loss. We set the weight of L1 to 1.
